src/main/java/io/cryostat/net/reports/ActiveRecordingReportCache.java

```diff
@@ -52,41 +53,50 @@
 
 import io.cryostat.core.log.Logger;
 import io.cryostat.core.sys.FileSystem;
+import io.cryostat.jmc.serialization.HyperlinkedSerializableRecordingDescriptor;
+import io.cryostat.messaging.notifications.Notification;
+import io.cryostat.messaging.notifications.NotificationListener;
+import io.cryostat.messaging.notifications.NotificationSource;
 import io.cryostat.net.ConnectionDescriptor;
 import io.cryostat.net.TargetConnectionManager;
 
 import com.github.benmanes.caffeine.cache.Caffeine;
 import com.github.benmanes.caffeine.cache.LoadingCache;
 import com.github.benmanes.caffeine.cache.Scheduler;
 
-class ActiveRecordingReportCache {
+class ActiveRecordingReportCache implements NotificationListener<Map<String, Object>> {
+    private static final String STOP_NOTIFICATION_CATEGORY = "ActiveRecordingStopped";
```

Comment:
This constant is already defined in the `RecordingTargetHelper` class, so this one should be removed and replaced with just references to that original definition.